### PR TITLE
crypto alpha-38 test

### PR DESCRIPTION
**Pull request by Jack-Works**

### CODE REVIEW

File: src/crypto/__tests__/crypto-alpha-38-test.ts [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+    makeTypedMessage,
+    TypedMessageComplex,
+    TypedMessageUnknown,
+} from '../../extension/background-script/CryptoServices/utils'
+import { derive_AES_GCM_256_Key_From_PBKDF2, import_PBKDF2_Key } from '../../utils/crypto.subtle'
+import { encodeText, encodeArrayBuffer, decodeText } from '../../utils/type-transform/String-ArrayBuffer'
+import { CryptoKeyToJsonWebKey } from '../../utils/type-transform/CryptoKey-JsonWebKey'
+import { recover_ECDH_256k1_KeyPair_ByMnemonicWord } from '../../utils/mnemonic-code'
+
+function helper(x: any) {
```
Comment (yisiliu) [inline src/crypto/__tests__/crypto-alpha-38-test.ts]: Better change the function name - helper doesn't make any sense.

Comment (Jack-Works) [reply src/crypto/__tests__/crypto-alpha-38-test.ts]: it's a local function, not exported

File: src/plugins/Wallet/web3.ts [hunk 2]
```diff
@@ -34,8 +34,12 @@ export const resetWallet = async () => {
 currentEthereumNetworkSettings.addListener(resetProvider)
 PluginMessageCenter.on('maskbook.wallets.reset', resetWallet)
 
-resetWallet()
-resetProvider()
+setTimeout(() => {
```
Comment (guanbinrui) [inline src/plugins/Wallet/web3.ts]: Why a `setTimeout` here?

Comment (Jack-Works) [reply src/plugins/Wallet/web3.ts]: because it will crash the test

**Pull request merged**